src/base/bittorrent/bandwidthscheduler.cpp

```diff
@@ -75,20 +75,34 @@ bool BandwidthScheduler::isTimeForAlternative() const
     {
         switch (schedulerDays)
         {
-        case EVERY_DAY:
+        case Scheduler::Days::EveryDay:
             alternative = !alternative;
             break;
-        case WEEK_ENDS:
-            if ((day == 6) || (day == 7))
+        case Scheduler::Days::Monday:
+        case Scheduler::Days::Tuesday:
+        case Scheduler::Days::Wednesday:
+        case Scheduler::Days::Thursday:
+        case Scheduler::Days::Friday:
+        case Scheduler::Days::Saturday:
+        case Scheduler::Days::Sunday:
+            {
+                const int offset = static_cast<int>(Scheduler::Days::Monday) - 1;
```

Comment:
The thing I talked before is mainly to avoid this offset.

Reply:
I understand but see appcontroller.cpp, the webAPI is using the integer representation, i.e. changing enum order will break it.

Reply:
We can adjust the values in `appcontroller.cpp` to keep it compatible. I agree, it looks like moving "using offset" from one place in the code to another, so I don't insist.
>IMO the breakage is unnecessary

I agree.
However, it would be better if API used "meaningful" enumeration member names instead of "meaningless" numbers.
But we can accumulate such not strictly required changes and apply them in a batch.

Reply:
IMO better leave it for later. It (webapi, gui) is not the main goal of this PR..

Reply:
>IMO better leave it for later. They (webapi, gui) are not the main goal of this PR..

Ok.